apps/emqx_authentication/data/user-credentials.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
```
[
    {
        "user_id": "myuser",
        "password_hash": "mypasshash",
        "salt": "salt"
    }
]
```
This format is better, it can support more fields.